src/plugins/intel_cpu/src/infer_request.cpp

```diff
@@ -3,41 +3,85 @@
 //
 
 #include "infer_request.h"
-#include "dnnl_extension_utils.h"
-#include <vector>
-#include <string>
-#include <map>
-#include <blob_factory.hpp>
-#include "nodes/concat.h"
-#include "nodes/split.h"
-#include <ie_compound_blob.h>
+
+#include <debug.h>
 #include <ie_common.h>
-#include "exec_network.h"
+
+#include <blob_factory.hpp>
+#include <ie_ngraph_utils.hpp>
+#include <map>
+#include <string>
+#include <transformations/utils/utils.hpp>
```

Comment:
please, use "" for OpenVINO headers. <> are used only for external ones

Reply:
ok, will do it.
Actually, I just changed the header file order due to clang-format and still keep the original header file format.